store/storetest/post_store.go

```diff
@@ -1374,7 +1374,7 @@ func testPostStoreGetFlaggedPostsForTeam(t *testing.T, ss store.Store, s SqlSupp
 	o5 = (<-ss.Post().Save(o5)).Data.(*model.Post)
 	time.Sleep(2 * time.Millisecond)
 
-	r1 := (<-ss.Post().GetFlaggedPosts(o1.ChannelId, 0, 2)).Data.(*model.PostList)
+	r1, _ := ss.Post().GetFlaggedPosts(o1.ChannelId, 0, 2)
```

Comment:
Please see my comment in your other PR: https://github.com/mattermost/mattermost-server/pull/10996#discussion_r288390445